utils/rsaencryption/rsa_encryption.go

```diff
@@ -59,18 +62,48 @@ func DecodeKey(sk *rsa.PrivateKey, hash []byte) ([]byte, error) {
 // ConvertPemToPrivateKey return rsa private key from secret key
 func ConvertPemToPrivateKey(skPem string) (*rsa.PrivateKey, error) {
 	block, _ := pem.Decode([]byte(skPem))
-	// TODO: resolve deprecation https://github.com/golang/go/issues/8860
-	enc := x509.IsEncryptedPEMBlock(block) //nolint
+	if block == nil {
+		return nil, errors.New("Failed to decode PEM block")
```

Comment:
[Error strings should not be capitalized.]("github.com/pkg/errors") Also, we can remove "failed to" to have a log like "failed to do X: Y: Z: error" instead of "failed to do X: failed to do Y: failed to do Z: error"  

```suggestion
		return nil, errors.New("decode PEM block")
```